include/pbbam/MD5.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2014-2015, Pacific Biosciences of California, Inc.
```

Comment:
You can put the license into a `LICENSE` file. That's SOP in GitHub. Then you don't have to include it in every file. Just FYI.